### ISSUE TITLE
Food truck intent improvements - Query response time

### ISSUE THREAD

**Issue by thiteixeira** [codeforboston/voiceapp311#220]
Currently, the food truck intent gets the user address, queries for all of the available food trucks, then calculates the distance between the user address and each of the food trucks, which has a high response time.
To reduce the delay, we implemented a hard limit of 5 food trucks, i.e., when the number of food within the desired distance reaches 5, we stop calculating the distance to the other ones.

We can pass the user address to the ArcGIS server and get the features that match our query, saving a bunch of time.

The query parameters are the following:
```
//distance:.5
//units:esriSRUnit_StatuteMile
inSR:4326
outSR:4326
outfields:*
//geometry:-71.073092022067257,42.336685825960636
//geometryType:esriGeometryPoint
f:json
```

**Comment by thiteixeira** [comment]
The code below currently returns
```
Cannot perform query. Invalid query parameters.
'geometry' parameter is invalid
```

```python
food_trucks_url = 'https://services.arcgis.com/sFnw0xNflSi8J0uh/arcgis/rest/services/food_trucks_schedule/FeatureServer/0'
address = geocode_address('120 Boylston St')
print(address)  # {'x': -71.06570492353856, 'y': 42.35243160170822}

QUERY = { 'where': '1=1', 'out_sr': '4326', 'distance': '0.5', 'units': 'esriSRUnit_StatuteMile', 'geometry': '-71.06570492353856, 42.35243160170822' }

# this will return a list of all available food trucks within a half mile from the address
trucks = get_feature_location(food_trucks_url, QUERY)
trucks_today = []
for t in trucks:
  if(t['attributes']['Day'] == DAY and t['attributes']['Time'] == 'Lunch'):
    trucks_today.append(t)

pprint(trucks_today)    
```

I will investigate more later this week.

**Comment by A-Camp** [comment]
I got it working during the Hackathon (with @thiteixeira's help) using the following pattern:
```
QUERY = {
    "distance":"1",
    "inSR":"4326",
    "outSR":"4326",
    "where":"1=1",
    "f":"json",
    "outfields":"*",
    "units":"esriSRUnit_StatuteMile",
    "geometry":"-71.06570492353856, 42.35243160170822",
    "geometryType":"esriGeometryPoint"
}
```

PR request can be found at - https://github.com/codeforboston/voiceapp311/pull/225

**Comment by jmartini** [comment]
Closed with #225
